contracts/tests/clarigen-types.ts

```diff
@@ -7,6 +7,85 @@ import type {
 } from "@clarigen/core";
 
 export const contracts = {
```

Comment:
What does all this do?

Reply:
Part of the Clarigen pnpm setup @hstove introduced. It easily allows us to generate TS types from Clarity contracts to use them in tests.